src/down_std.ml

```diff
@@ -267,17 +267,23 @@ module Dir = struct
   let config () = match Env.get "XDG_CONFIG_HOME" with
   | Some h -> Ok h
   | None ->
-      match if Sys.win32 then Env.get "%APPDATA%" else None with
+      match if Sys.win32 then Env.get "APPDATA" else None with
       | Some h -> Ok h
       | None ->
           match Env.get "HOME" with
           | Some h -> Ok (Filename.concat h ".config")
           | None -> Error "Could not determine a user configuration directory"
 
-  let mkdir_win32 dir = ["mkdir"; dir]
-  let mkdir_posix dir = ["mkdir"; "-p"; dir]
-  let mkdir = if Sys.win32 then mkdir_win32 else mkdir_posix
-  let create dir = Result.map_error snd @@ cmd_run (mkdir dir)
+  let mkdir_p dir =
```

Comment:
`mkdir -p` does not exist in `cmd.exe`. But `Sys.mkdir` exists in the standard library, so perhaps some version of the code below could be used instead.